.travis.yml

```diff
@@ -16,5 +16,8 @@ before_install:
   - bin/buildout -N -t 3 annotate
 install:
   - bin/buildout -N -t 3
+before_script:
+  - bin/pip install flake8
 script:
+  - bin/flake8 --config travis_ci_flake8.cfg src/
```

Comment:
it looks like Travis CI currently doesn't run on pull requests, so this change is not tested

Reply:
At least we can run it manually against the configuration, so no worries.